_docs/concepts/traffic-management/request-routing.md

```diff
@@ -12,11 +12,15 @@ This page describes how requests are routed between services in an Istio service
 
 ## Service model and service versions.
 
-As described in [Istio-Manager](./manager.html), the canonical representation of services in a particular mesh is maintained by the Istio-Manager service. The Istio model of a service is independent of how it is represented in the underlying platform (Kubernetes, Mesos, Cloud Foundry,
+As described in [Istio-Manager](./manager.html), the canonical representation
```

Comment:
"the manager" sounds odd there, as we've never used that term to describe Istio-Manager - I know the duplication of Istio-Manager sounds a bit clunky but I think the alternatives are worse.